/*
 * This file was automatically generated by EvoSuite
 * Wed May 10 16:53:05 GMT 2023
 */

package Model.Exceptions;

import org.junit.Test;
import static org.junit.Assert.*;
import Model.Exceptions.LogAlreadyExistsException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class LogAlreadyExistsException_ESTest extends LogAlreadyExistsException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      LogAlreadyExistsException logAlreadyExistsException0 = new LogAlreadyExistsException("IJ,ckLE#tL6(=Rz");
  }
}
